app/Models/Comment.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Comment extends Model
```

Comment:
fillable dùng để làm gì?

Reply:
Mặc định thì các thuộc tính trong Eloquent Model được bảo vệ khỏi lỗ hổng Mass Assignment. User sẽ không thể truyển thuộc tính không có trong $fillable khi gửi HTTP Request.

Reply:
Người dùng vẫn có thể gửi các trường ko nằm trong fillable trong HTTP Request.

Nó chỉ chặn việc đổ dữ liệu hàng loạt đối với các giá trị ko nằm trong **$fillable** (hoặc nằm trong **$guard**) khi ta sử dụng các hàm như **create()**, **update()**. Nếu muốn ta hoàn toàn có thể lấy dữ liệu đó ra manually như sau:

$model->attribute_name = $request->attribute;
$model->save();

Reply:
done